sentinel5dl/__main__.py

```diff
@@ -141,8 +142,22 @@ def main():
             Example: 2019-09-17T23:59:59.999Z'''
     )
 
+    parser.add_argument(
+        '--use-certifi',
+        default='false',
```

Comment:
Don't set this. `action='store_true'` already comes with a default which is the boolean value `False`. You are overwriting this with the string `"false"` making this always truthy¹.

```suggestion
```

---

**\*¹** A truthy value means that it evaluates to `True`. A non-emty string evaluates to `True`, thus `"false"` evaluates to `True`:

```py
>>> bool('false')
True
```